app/helpers/task_view_helper.rb

```diff
@@ -0,0 +1,17 @@
+module TaskViewHelper
+  def task_view_header(choice)
+    case choice&.task_view_group
```

Comment:
Very minor: Given this is a bunch of static strings we could just use a hash and a default return value with `fetch`.